pom.xml

```diff
@@ -26,6 +26,11 @@
             <artifactId>junit-jupiter-api</artifactId>
             <version>5.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>antlr</groupId>
+            <artifactId>antlr</artifactId>
+            <version>2.7.2</version>
+        </dependency>
```

Comment:
*ANTLR to narzędzie służące do tworzenia kompilatorów oraz translatorów z opisu gramatyki zawierającego akcje w języku Java, C++, C# lub Python.* co tu sie?